recipes/calibrated-explanations/LICENSE

```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```

Comment:
It looks like this license is already included in the source distribution, so you don't need to add it again here in the recipe.